sonarts-core/tests/rules/noDuplicateStringRule/noDuplicateStringRule.lint.tsx

```diff
@@ -1,3 +1,53 @@
+
+/// <reference lib="./module-name-long" />
+/// <reference lib="./module-name-long" />
+/// <reference lib="./module-name-long" />
+/// <reference lib="./module-name-long" />
+
+// taken from
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import
+import defaultExport from "module-name-long";
```

Comment:
@maximelkin do we really need such exhaustive testing of all import and export variants? I would be ok, for couple import variants and couple export.